src/transform/WidenImage.js

```diff
@@ -1,30 +1,99 @@
 import './WidenImage.css'
 import elementUtilities from './ElementUtilities'
 
+/**
+ * Gets array of ancestors of element which need widening.
+ * @param  {!HTMLElement} element
+ * @return {!Array.<HTMLElement>} Zero length array is returned if no elements should be widened.
+ */
+const ancestorsToWiden = element => {
+  const widenThese = []
+  let el = element
+  while (el.parentNode) {
+    el = el.parentNode
+    // No need to walk above 'content_block'.
+    if (el.classList.contains('content_block')) {
+      break
+    }
+    widenThese.push(el)
+  }
+  return widenThese
+}
+
+/**
+ * Sets style value.
+ * @param {HTMLElement.style} style
+ * @param {!string} key
+ * @param {*} value
+ * @return {void}
+ */
+const updateStyleValue = (style, key, value) => {
+  style[key] = value
+}
+
+/**
+ * Sets style value only if value for given key already exists.
+ * @param {HTMLElement.style} style
+ * @param {!string} key
+ * @param {*} value
+ * @return {void}
+ */
+const updateExistingStyleValue = (style, key, value) => {
+  const existingValue = style[key]
+  if (!existingValue) {
+    return
+  }
+  updateStyleValue(style, key, value)
+}
+
+/**
+ * Array of arrays of image widening CSS key/value pairs.
+ * @type {Array}
+ */
+const styleWideningKeysAndValues = [
+  ['width', '100%'],
```

Comment:
Instead of using an array of arrays with the spread operator, this code might be hardier if it's changed to an array of objects, each with key and value members, or even better, just an object since order doesn't matter:

```js
const overrides = {
  width: '100%',
  height: 'auto',
  maxWidth: '100%',
  float: 'none'
}
```

Reply:
Ah, ya that makes sense. No real need for spread fanciness...

Reply:
Fixed!